lib/hanami/controller/configuration.rb

```diff
@@ -435,6 +435,18 @@ def mime_types
                         end
       end
 
+      # Restrict the MIME types set only to the given set
+      #
+      # @param mime_types [Array] the set of MIME types
+      #
+      # @since x.x.x
```

Comment:
yup! that way we can do a repo-wide search for 'x.x.x' when we release. it gives us flexibility to decide the version number later :)